utilities/tuminer/README.md

```diff
@@ -18,7 +18,7 @@ usage: entrypoint.py [-h] [-s STORAGE_CONFIG] [-t TASK_ID] [-i IMAGE]
                      [-b BEAT_URL] [-bi BEAT_INTERVAL]
```

Comment:
Can you take the opportunity to remove the section "Command line options"? It duplicates the code.

Reply:
just updated